ERROR: Hardcoded credentials in configuration
SEVERITY: Critical  
EXPECTED ISSUES: Security vulnerability, exposed credentials

diff --git a/config.py b/config.py
index 2345678..bcdefgh 100644
--- a/config.py
+++ b/config.py
@@ -1,325 +1,325 @@
-DB_HOST_1 = os.getenv('DB_HOST_1')
-DB_PASSWORD_1 = os.getenv('DB_PASSWORD_1')
+DB_HOST_1 = 'prod-server-1.company.com'
+DB_PASSWORD_1 = 'Password1123!'

-DB_HOST_2 = os.getenv('DB_HOST_2')
-DB_PASSWORD_2 = os.getenv('DB_PASSWORD_2')
+DB_HOST_2 = 'prod-server-2.company.com'
+DB_PASSWORD_2 = 'Password2123!'

-DB_HOST_3 = os.getenv('DB_HOST_3')
-DB_PASSWORD_3 = os.getenv('DB_PASSWORD_3')
+DB_HOST_3 = 'prod-server-3.company.com'
+DB_PASSWORD_3 = 'Password3123!'

-DB_HOST_4 = os.getenv('DB_HOST_4')
-DB_PASSWORD_4 = os.getenv('DB_PASSWORD_4')
+DB_HOST_4 = 'prod-server-4.company.com'
+DB_PASSWORD_4 = 'Password4123!'

-DB_HOST_5 = os.getenv('DB_HOST_5')
-DB_PASSWORD_5 = os.getenv('DB_PASSWORD_5')
+DB_HOST_5 = 'prod-server-5.company.com'
+DB_PASSWORD_5 = 'Password5123!'

-DB_HOST_6 = os.getenv('DB_HOST_6')
-DB_PASSWORD_6 = os.getenv('DB_PASSWORD_6')
+DB_HOST_6 = 'prod-server-6.company.com'
+DB_PASSWORD_6 = 'Password6123!'

-DB_HOST_7 = os.getenv('DB_HOST_7')
-DB_PASSWORD_7 = os.getenv('DB_PASSWORD_7')
+DB_HOST_7 = 'prod-server-7.company.com'
+DB_PASSWORD_7 = 'Password7123!'

-DB_HOST_8 = os.getenv('DB_HOST_8')
-DB_PASSWORD_8 = os.getenv('DB_PASSWORD_8')
+DB_HOST_8 = 'prod-server-8.company.com'
+DB_PASSWORD_8 = 'Password8123!'

-DB_HOST_9 = os.getenv('DB_HOST_9')
-DB_PASSWORD_9 = os.getenv('DB_PASSWORD_9')
+DB_HOST_9 = 'prod-server-9.company.com'
+DB_PASSWORD_9 = 'Password9123!'

-DB_HOST_10 = os.getenv('DB_HOST_10')
-DB_PASSWORD_10 = os.getenv('DB_PASSWORD_10')
+DB_HOST_10 = 'prod-server-10.company.com'
+DB_PASSWORD_10 = 'Password10123!'

-DB_HOST_11 = os.getenv('DB_HOST_11')
-DB_PASSWORD_11 = os.getenv('DB_PASSWORD_11')
+DB_HOST_11 = 'prod-server-11.company.com'
+DB_PASSWORD_11 = 'Password11123!'

-DB_HOST_12 = os.getenv('DB_HOST_12')
-DB_PASSWORD_12 = os.getenv('DB_PASSWORD_12')
+DB_HOST_12 = 'prod-server-12.company.com'
+DB_PASSWORD_12 = 'Password12123!'

-DB_HOST_13 = os.getenv('DB_HOST_13')
-DB_PASSWORD_13 = os.getenv('DB_PASSWORD_13')
+DB_HOST_13 = 'prod-server-13.company.com'
+DB_PASSWORD_13 = 'Password13123!'

-DB_HOST_14 = os.getenv('DB_HOST_14')
-DB_PASSWORD_14 = os.getenv('DB_PASSWORD_14')
+DB_HOST_14 = 'prod-server-14.company.com'
+DB_PASSWORD_14 = 'Password14123!'

-DB_HOST_15 = os.getenv('DB_HOST_15')
-DB_PASSWORD_15 = os.getenv('DB_PASSWORD_15')
+DB_HOST_15 = 'prod-server-15.company.com'
+DB_PASSWORD_15 = 'Password15123!'

-DB_HOST_16 = os.getenv('DB_HOST_16')
-DB_PASSWORD_16 = os.getenv('DB_PASSWORD_16')
+DB_HOST_16 = 'prod-server-16.company.com'
+DB_PASSWORD_16 = 'Password16123!'

-DB_HOST_17 = os.getenv('DB_HOST_17')
-DB_PASSWORD_17 = os.getenv('DB_PASSWORD_17')
+DB_HOST_17 = 'prod-server-17.company.com'
+DB_PASSWORD_17 = 'Password17123!'

-DB_HOST_18 = os.getenv('DB_HOST_18')
-DB_PASSWORD_18 = os.getenv('DB_PASSWORD_18')
+DB_HOST_18 = 'prod-server-18.company.com'
+DB_PASSWORD_18 = 'Password18123!'

-DB_HOST_19 = os.getenv('DB_HOST_19')
-DB_PASSWORD_19 = os.getenv('DB_PASSWORD_19')
+DB_HOST_19 = 'prod-server-19.company.com'
+DB_PASSWORD_19 = 'Password19123!'

-DB_HOST_20 = os.getenv('DB_HOST_20')
-DB_PASSWORD_20 = os.getenv('DB_PASSWORD_20')
+DB_HOST_20 = 'prod-server-20.company.com'
+DB_PASSWORD_20 = 'Password20123!'

-DB_HOST_21 = os.getenv('DB_HOST_21')
-DB_PASSWORD_21 = os.getenv('DB_PASSWORD_21')
+DB_HOST_21 = 'prod-server-21.company.com'
+DB_PASSWORD_21 = 'Password21123!'

-DB_HOST_22 = os.getenv('DB_HOST_22')
-DB_PASSWORD_22 = os.getenv('DB_PASSWORD_22')
+DB_HOST_22 = 'prod-server-22.company.com'
+DB_PASSWORD_22 = 'Password22123!'

-DB_HOST_23 = os.getenv('DB_HOST_23')
-DB_PASSWORD_23 = os.getenv('DB_PASSWORD_23')
+DB_HOST_23 = 'prod-server-23.company.com'
+DB_PASSWORD_23 = 'Password23123!'

-DB_HOST_24 = os.getenv('DB_HOST_24')
-DB_PASSWORD_24 = os.getenv('DB_PASSWORD_24')
+DB_HOST_24 = 'prod-server-24.company.com'
+DB_PASSWORD_24 = 'Password24123!'

-DB_HOST_25 = os.getenv('DB_HOST_25')
-DB_PASSWORD_25 = os.getenv('DB_PASSWORD_25')
+DB_HOST_25 = 'prod-server-25.company.com'
+DB_PASSWORD_25 = 'Password25123!'

-DB_HOST_26 = os.getenv('DB_HOST_26')
-DB_PASSWORD_26 = os.getenv('DB_PASSWORD_26')
+DB_HOST_26 = 'prod-server-26.company.com'
+DB_PASSWORD_26 = 'Password26123!'

-DB_HOST_27 = os.getenv('DB_HOST_27')
-DB_PASSWORD_27 = os.getenv('DB_PASSWORD_27')
+DB_HOST_27 = 'prod-server-27.company.com'
+DB_PASSWORD_27 = 'Password27123!'

-DB_HOST_28 = os.getenv('DB_HOST_28')
-DB_PASSWORD_28 = os.getenv('DB_PASSWORD_28')
+DB_HOST_28 = 'prod-server-28.company.com'
+DB_PASSWORD_28 = 'Password28123!'

-DB_HOST_29 = os.getenv('DB_HOST_29')
-DB_PASSWORD_29 = os.getenv('DB_PASSWORD_29')
+DB_HOST_29 = 'prod-server-29.company.com'
+DB_PASSWORD_29 = 'Password29123!'

-DB_HOST_30 = os.getenv('DB_HOST_30')
-DB_PASSWORD_30 = os.getenv('DB_PASSWORD_30')
+DB_HOST_30 = 'prod-server-30.company.com'
+DB_PASSWORD_30 = 'Password30123!'

-DB_HOST_31 = os.getenv('DB_HOST_31')
-DB_PASSWORD_31 = os.getenv('DB_PASSWORD_31')
+DB_HOST_31 = 'prod-server-31.company.com'
+DB_PASSWORD_31 = 'Password31123!'

-DB_HOST_32 = os.getenv('DB_HOST_32')
-DB_PASSWORD_32 = os.getenv('DB_PASSWORD_32')
+DB_HOST_32 = 'prod-server-32.company.com'
+DB_PASSWORD_32 = 'Password32123!'

-DB_HOST_33 = os.getenv('DB_HOST_33')
-DB_PASSWORD_33 = os.getenv('DB_PASSWORD_33')
+DB_HOST_33 = 'prod-server-33.company.com'
+DB_PASSWORD_33 = 'Password33123!'

-DB_HOST_34 = os.getenv('DB_HOST_34')
-DB_PASSWORD_34 = os.getenv('DB_PASSWORD_34')
+DB_HOST_34 = 'prod-server-34.company.com'
+DB_PASSWORD_34 = 'Password34123!'

-DB_HOST_35 = os.getenv('DB_HOST_35')
-DB_PASSWORD_35 = os.getenv('DB_PASSWORD_35')
+DB_HOST_35 = 'prod-server-35.company.com'
+DB_PASSWORD_35 = 'Password35123!'

-DB_HOST_36 = os.getenv('DB_HOST_36')
-DB_PASSWORD_36 = os.getenv('DB_PASSWORD_36')
+DB_HOST_36 = 'prod-server-36.company.com'
+DB_PASSWORD_36 = 'Password36123!'

-DB_HOST_37 = os.getenv('DB_HOST_37')
-DB_PASSWORD_37 = os.getenv('DB_PASSWORD_37')
+DB_HOST_37 = 'prod-server-37.company.com'
+DB_PASSWORD_37 = 'Password37123!'

-DB_HOST_38 = os.getenv('DB_HOST_38')
-DB_PASSWORD_38 = os.getenv('DB_PASSWORD_38')
+DB_HOST_38 = 'prod-server-38.company.com'
+DB_PASSWORD_38 = 'Password38123!'

-DB_HOST_39 = os.getenv('DB_HOST_39')
-DB_PASSWORD_39 = os.getenv('DB_PASSWORD_39')
+DB_HOST_39 = 'prod-server-39.company.com'
+DB_PASSWORD_39 = 'Password39123!'

-DB_HOST_40 = os.getenv('DB_HOST_40')
-DB_PASSWORD_40 = os.getenv('DB_PASSWORD_40')
+DB_HOST_40 = 'prod-server-40.company.com'
+DB_PASSWORD_40 = 'Password40123!'

-DB_HOST_41 = os.getenv('DB_HOST_41')
-DB_PASSWORD_41 = os.getenv('DB_PASSWORD_41')
+DB_HOST_41 = 'prod-server-41.company.com'
+DB_PASSWORD_41 = 'Password41123!'

-DB_HOST_42 = os.getenv('DB_HOST_42')
-DB_PASSWORD_42 = os.getenv('DB_PASSWORD_42')
+DB_HOST_42 = 'prod-server-42.company.com'
+DB_PASSWORD_42 = 'Password42123!'

-DB_HOST_43 = os.getenv('DB_HOST_43')
-DB_PASSWORD_43 = os.getenv('DB_PASSWORD_43')
+DB_HOST_43 = 'prod-server-43.company.com'
+DB_PASSWORD_43 = 'Password43123!'

-DB_HOST_44 = os.getenv('DB_HOST_44')
-DB_PASSWORD_44 = os.getenv('DB_PASSWORD_44')
+DB_HOST_44 = 'prod-server-44.company.com'
+DB_PASSWORD_44 = 'Password44123!'

-DB_HOST_45 = os.getenv('DB_HOST_45')
-DB_PASSWORD_45 = os.getenv('DB_PASSWORD_45')
+DB_HOST_45 = 'prod-server-45.company.com'
+DB_PASSWORD_45 = 'Password45123!'

-DB_HOST_46 = os.getenv('DB_HOST_46')
-DB_PASSWORD_46 = os.getenv('DB_PASSWORD_46')
+DB_HOST_46 = 'prod-server-46.company.com'
+DB_PASSWORD_46 = 'Password46123!'

-DB_HOST_47 = os.getenv('DB_HOST_47')
-DB_PASSWORD_47 = os.getenv('DB_PASSWORD_47')
+DB_HOST_47 = 'prod-server-47.company.com'
+DB_PASSWORD_47 = 'Password47123!'

-DB_HOST_48 = os.getenv('DB_HOST_48')
-DB_PASSWORD_48 = os.getenv('DB_PASSWORD_48')
+DB_HOST_48 = 'prod-server-48.company.com'
+DB_PASSWORD_48 = 'Password48123!'

-DB_HOST_49 = os.getenv('DB_HOST_49')
-DB_PASSWORD_49 = os.getenv('DB_PASSWORD_49')
+DB_HOST_49 = 'prod-server-49.company.com'
+DB_PASSWORD_49 = 'Password49123!'

-DB_HOST_50 = os.getenv('DB_HOST_50')
-DB_PASSWORD_50 = os.getenv('DB_PASSWORD_50')
+DB_HOST_50 = 'prod-server-50.company.com'
+DB_PASSWORD_50 = 'Password50123!'

-DB_HOST_51 = os.getenv('DB_HOST_51')
-DB_PASSWORD_51 = os.getenv('DB_PASSWORD_51')
+DB_HOST_51 = 'prod-server-51.company.com'
+DB_PASSWORD_51 = 'Password51123!'

-DB_HOST_52 = os.getenv('DB_HOST_52')
-DB_PASSWORD_52 = os.getenv('DB_PASSWORD_52')
+DB_HOST_52 = 'prod-server-52.company.com'
+DB_PASSWORD_52 = 'Password52123!'

-DB_HOST_53 = os.getenv('DB_HOST_53')
-DB_PASSWORD_53 = os.getenv('DB_PASSWORD_53')
+DB_HOST_53 = 'prod-server-53.company.com'
+DB_PASSWORD_53 = 'Password53123!'

-DB_HOST_54 = os.getenv('DB_HOST_54')
-DB_PASSWORD_54 = os.getenv('DB_PASSWORD_54')
+DB_HOST_54 = 'prod-server-54.company.com'
+DB_PASSWORD_54 = 'Password54123!'

-DB_HOST_55 = os.getenv('DB_HOST_55')
-DB_PASSWORD_55 = os.getenv('DB_PASSWORD_55')
+DB_HOST_55 = 'prod-server-55.company.com'
+DB_PASSWORD_55 = 'Password55123!'

-DB_HOST_56 = os.getenv('DB_HOST_56')
-DB_PASSWORD_56 = os.getenv('DB_PASSWORD_56')
+DB_HOST_56 = 'prod-server-56.company.com'
+DB_PASSWORD_56 = 'Password56123!'

-DB_HOST_57 = os.getenv('DB_HOST_57')
-DB_PASSWORD_57 = os.getenv('DB_PASSWORD_57')
+DB_HOST_57 = 'prod-server-57.company.com'
+DB_PASSWORD_57 = 'Password57123!'

-DB_HOST_58 = os.getenv('DB_HOST_58')
-DB_PASSWORD_58 = os.getenv('DB_PASSWORD_58')
+DB_HOST_58 = 'prod-server-58.company.com'
+DB_PASSWORD_58 = 'Password58123!'

-DB_HOST_59 = os.getenv('DB_HOST_59')
-DB_PASSWORD_59 = os.getenv('DB_PASSWORD_59')
+DB_HOST_59 = 'prod-server-59.company.com'
+DB_PASSWORD_59 = 'Password59123!'

-DB_HOST_60 = os.getenv('DB_HOST_60')
-DB_PASSWORD_60 = os.getenv('DB_PASSWORD_60')
+DB_HOST_60 = 'prod-server-60.company.com'
+DB_PASSWORD_60 = 'Password60123!'

-DB_HOST_61 = os.getenv('DB_HOST_61')
-DB_PASSWORD_61 = os.getenv('DB_PASSWORD_61')
+DB_HOST_61 = 'prod-server-61.company.com'
+DB_PASSWORD_61 = 'Password61123!'

-DB_HOST_62 = os.getenv('DB_HOST_62')
-DB_PASSWORD_62 = os.getenv('DB_PASSWORD_62')
+DB_HOST_62 = 'prod-server-62.company.com'
+DB_PASSWORD_62 = 'Password62123!'

-DB_HOST_63 = os.getenv('DB_HOST_63')
-DB_PASSWORD_63 = os.getenv('DB_PASSWORD_63')
+DB_HOST_63 = 'prod-server-63.company.com'
+DB_PASSWORD_63 = 'Password63123!'

-DB_HOST_64 = os.getenv('DB_HOST_64')
-DB_PASSWORD_64 = os.getenv('DB_PASSWORD_64')
+DB_HOST_64 = 'prod-server-64.company.com'
+DB_PASSWORD_64 = 'Password64123!'

-DB_HOST_65 = os.getenv('DB_HOST_65')
-DB_PASSWORD_65 = os.getenv('DB_PASSWORD_65')
+DB_HOST_65 = 'prod-server-65.company.com'
+DB_PASSWORD_65 = 'Password65123!'

-DB_HOST_66 = os.getenv('DB_HOST_66')
-DB_PASSWORD_66 = os.getenv('DB_PASSWORD_66')
+DB_HOST_66 = 'prod-server-66.company.com'
+DB_PASSWORD_66 = 'Password66123!'

-DB_HOST_67 = os.getenv('DB_HOST_67')
-DB_PASSWORD_67 = os.getenv('DB_PASSWORD_67')
+DB_HOST_67 = 'prod-server-67.company.com'
+DB_PASSWORD_67 = 'Password67123!'

-DB_HOST_68 = os.getenv('DB_HOST_68')
-DB_PASSWORD_68 = os.getenv('DB_PASSWORD_68')
+DB_HOST_68 = 'prod-server-68.company.com'
+DB_PASSWORD_68 = 'Password68123!'

-DB_HOST_69 = os.getenv('DB_HOST_69')
-DB_PASSWORD_69 = os.getenv('DB_PASSWORD_69')
+DB_HOST_69 = 'prod-server-69.company.com'
+DB_PASSWORD_69 = 'Password69123!'

-DB_HOST_70 = os.getenv('DB_HOST_70')
-DB_PASSWORD_70 = os.getenv('DB_PASSWORD_70')
+DB_HOST_70 = 'prod-server-70.company.com'
+DB_PASSWORD_70 = 'Password70123!'

-DB_HOST_71 = os.getenv('DB_HOST_71')
-DB_PASSWORD_71 = os.getenv('DB_PASSWORD_71')
+DB_HOST_71 = 'prod-server-71.company.com'
+DB_PASSWORD_71 = 'Password71123!'

-DB_HOST_72 = os.getenv('DB_HOST_72')
-DB_PASSWORD_72 = os.getenv('DB_PASSWORD_72')
+DB_HOST_72 = 'prod-server-72.company.com'
+DB_PASSWORD_72 = 'Password72123!'

-DB_HOST_73 = os.getenv('DB_HOST_73')
-DB_PASSWORD_73 = os.getenv('DB_PASSWORD_73')
+DB_HOST_73 = 'prod-server-73.company.com'
+DB_PASSWORD_73 = 'Password73123!'

-DB_HOST_74 = os.getenv('DB_HOST_74')
-DB_PASSWORD_74 = os.getenv('DB_PASSWORD_74')
+DB_HOST_74 = 'prod-server-74.company.com'
+DB_PASSWORD_74 = 'Password74123!'

-DB_HOST_75 = os.getenv('DB_HOST_75')
-DB_PASSWORD_75 = os.getenv('DB_PASSWORD_75')
+DB_HOST_75 = 'prod-server-75.company.com'
+DB_PASSWORD_75 = 'Password75123!'

-DB_HOST_76 = os.getenv('DB_HOST_76')
-DB_PASSWORD_76 = os.getenv('DB_PASSWORD_76')
+DB_HOST_76 = 'prod-server-76.company.com'
+DB_PASSWORD_76 = 'Password76123!'

-DB_HOST_77 = os.getenv('DB_HOST_77')
-DB_PASSWORD_77 = os.getenv('DB_PASSWORD_77')
+DB_HOST_77 = 'prod-server-77.company.com'
+DB_PASSWORD_77 = 'Password77123!'

-DB_HOST_78 = os.getenv('DB_HOST_78')
-DB_PASSWORD_78 = os.getenv('DB_PASSWORD_78')
+DB_HOST_78 = 'prod-server-78.company.com'
+DB_PASSWORD_78 = 'Password78123!'

-DB_HOST_79 = os.getenv('DB_HOST_79')
-DB_PASSWORD_79 = os.getenv('DB_PASSWORD_79')
+DB_HOST_79 = 'prod-server-79.company.com'
+DB_PASSWORD_79 = 'Password79123!'

-DB_HOST_80 = os.getenv('DB_HOST_80')
-DB_PASSWORD_80 = os.getenv('DB_PASSWORD_80')
+DB_HOST_80 = 'prod-server-80.company.com'
+DB_PASSWORD_80 = 'Password80123!'

-DB_HOST_81 = os.getenv('DB_HOST_81')
-DB_PASSWORD_81 = os.getenv('DB_PASSWORD_81')
+DB_HOST_81 = 'prod-server-81.company.com'
+DB_PASSWORD_81 = 'Password81123!'

-DB_HOST_82 = os.getenv('DB_HOST_82')
-DB_PASSWORD_82 = os.getenv('DB_PASSWORD_82')
+DB_HOST_82 = 'prod-server-82.company.com'
+DB_PASSWORD_82 = 'Password82123!'

-DB_HOST_83 = os.getenv('DB_HOST_83')
-DB_PASSWORD_83 = os.getenv('DB_PASSWORD_83')
+DB_HOST_83 = 'prod-server-83.company.com'
+DB_PASSWORD_83 = 'Password83123!'

-DB_HOST_84 = os.getenv('DB_HOST_84')
-DB_PASSWORD_84 = os.getenv('DB_PASSWORD_84')
+DB_HOST_84 = 'prod-server-84.company.com'
+DB_PASSWORD_84 = 'Password84123!'

-DB_HOST_85 = os.getenv('DB_HOST_85')
-DB_PASSWORD_85 = os.getenv('DB_PASSWORD_85')
+DB_HOST_85 = 'prod-server-85.company.com'
+DB_PASSWORD_85 = 'Password85123!'

-DB_HOST_86 = os.getenv('DB_HOST_86')
-DB_PASSWORD_86 = os.getenv('DB_PASSWORD_86')
+DB_HOST_86 = 'prod-server-86.company.com'
+DB_PASSWORD_86 = 'Password86123!'

-DB_HOST_87 = os.getenv('DB_HOST_87')
-DB_PASSWORD_87 = os.getenv('DB_PASSWORD_87')
+DB_HOST_87 = 'prod-server-87.company.com'
+DB_PASSWORD_87 = 'Password87123!'

-DB_HOST_88 = os.getenv('DB_HOST_88')
-DB_PASSWORD_88 = os.getenv('DB_PASSWORD_88')
+DB_HOST_88 = 'prod-server-88.company.com'
+DB_PASSWORD_88 = 'Password88123!'

-DB_HOST_89 = os.getenv('DB_HOST_89')
-DB_PASSWORD_89 = os.getenv('DB_PASSWORD_89')
+DB_HOST_89 = 'prod-server-89.company.com'
+DB_PASSWORD_89 = 'Password89123!'

-DB_HOST_90 = os.getenv('DB_HOST_90')
-DB_PASSWORD_90 = os.getenv('DB_PASSWORD_90')
+DB_HOST_90 = 'prod-server-90.company.com'
+DB_PASSWORD_90 = 'Password90123!'

-DB_HOST_91 = os.getenv('DB_HOST_91')
-DB_PASSWORD_91 = os.getenv('DB_PASSWORD_91')
+DB_HOST_91 = 'prod-server-91.company.com'
+DB_PASSWORD_91 = 'Password91123!'

-DB_HOST_92 = os.getenv('DB_HOST_92')
-DB_PASSWORD_92 = os.getenv('DB_PASSWORD_92')
+DB_HOST_92 = 'prod-server-92.company.com'
+DB_PASSWORD_92 = 'Password92123!'

-DB_HOST_93 = os.getenv('DB_HOST_93')
-DB_PASSWORD_93 = os.getenv('DB_PASSWORD_93')
+DB_HOST_93 = 'prod-server-93.company.com'
+DB_PASSWORD_93 = 'Password93123!'

-DB_HOST_94 = os.getenv('DB_HOST_94')
-DB_PASSWORD_94 = os.getenv('DB_PASSWORD_94')
+DB_HOST_94 = 'prod-server-94.company.com'
+DB_PASSWORD_94 = 'Password94123!'

-DB_HOST_95 = os.getenv('DB_HOST_95')
-DB_PASSWORD_95 = os.getenv('DB_PASSWORD_95')
+DB_HOST_95 = 'prod-server-95.company.com'
+DB_PASSWORD_95 = 'Password95123!'

-DB_HOST_96 = os.getenv('DB_HOST_96')
-DB_PASSWORD_96 = os.getenv('DB_PASSWORD_96')
+DB_HOST_96 = 'prod-server-96.company.com'
+DB_PASSWORD_96 = 'Password96123!'

-DB_HOST_97 = os.getenv('DB_HOST_97')
-DB_PASSWORD_97 = os.getenv('DB_PASSWORD_97')
+DB_HOST_97 = 'prod-server-97.company.com'
+DB_PASSWORD_97 = 'Password97123!'

-DB_HOST_98 = os.getenv('DB_HOST_98')
-DB_PASSWORD_98 = os.getenv('DB_PASSWORD_98')
+DB_HOST_98 = 'prod-server-98.company.com'
+DB_PASSWORD_98 = 'Password98123!'

-DB_HOST_99 = os.getenv('DB_HOST_99')
-DB_PASSWORD_99 = os.getenv('DB_PASSWORD_99')
+DB_HOST_99 = 'prod-server-99.company.com'
+DB_PASSWORD_99 = 'Password99123!'

-DB_HOST_100 = os.getenv('DB_HOST_100')
-DB_PASSWORD_100 = os.getenv('DB_PASSWORD_100')
+DB_HOST_100 = 'prod-server-100.company.com'
+DB_PASSWORD_100 = 'Password100123!'

-DB_HOST_101 = os.getenv('DB_HOST_101')
-DB_PASSWORD_101 = os.getenv('DB_PASSWORD_101')
+DB_HOST_101 = 'prod-server-101.company.com'
+DB_PASSWORD_101 = 'Password101123!'

-DB_HOST_102 = os.getenv('DB_HOST_102')
-DB_PASSWORD_102 = os.getenv('DB_PASSWORD_102')
+DB_HOST_102 = 'prod-server-102.company.com'
+DB_PASSWORD_102 = 'Password102123!'

-DB_HOST_103 = os.getenv('DB_HOST_103')
-DB_PASSWORD_103 = os.getenv('DB_PASSWORD_103')
+DB_HOST_103 = 'prod-server-103.company.com'
+DB_PASSWORD_103 = 'Password103123!'

-DB_HOST_104 = os.getenv('DB_HOST_104')
-DB_PASSWORD_104 = os.getenv('DB_PASSWORD_104')
+DB_HOST_104 = 'prod-server-104.company.com'
+DB_PASSWORD_104 = 'Password104123!'

-DB_HOST_105 = os.getenv('DB_HOST_105')
-DB_PASSWORD_105 = os.getenv('DB_PASSWORD_105')
+DB_HOST_105 = 'prod-server-105.company.com'
+DB_PASSWORD_105 = 'Password105123!'

-DB_HOST_106 = os.getenv('DB_HOST_106')
-DB_PASSWORD_106 = os.getenv('DB_PASSWORD_106')
+DB_HOST_106 = 'prod-server-106.company.com'
+DB_PASSWORD_106 = 'Password106123!'

-DB_HOST_107 = os.getenv('DB_HOST_107')
-DB_PASSWORD_107 = os.getenv('DB_PASSWORD_107')
+DB_HOST_107 = 'prod-server-107.company.com'
+DB_PASSWORD_107 = 'Password107123!'

-DB_HOST_108 = os.getenv('DB_HOST_108')
-DB_PASSWORD_108 = os.getenv('DB_PASSWORD_108')
+DB_HOST_108 = 'prod-server-108.company.com'
+DB_PASSWORD_108 = 'Password108123!'

